Admin/AdminInterface.php

```diff
@@ -308,6 +308,21 @@ public function getIdParameter();
     public function hasRoute($name);
 
     /**
+     * Check the current request is given route or not.
+     *
+     * ```
+     * $this->isRoute('create'); // is create page?
+     * $this->isRoute('edit', 'some.admin.code'); // is some.admin.code admin's edit page?
+     * ```
+     *
+     * @param string $name
+     * @param string $adminCode
+     *
+     * @return bool
+     */
+    public function isRoute($name, $adminCode = null);
```

Comment:
When you add methods to in Interface, it is a BC break.

Please comment this line, so we can merge this earlier